### PR TITLE
Tp new thank you flow

### PR DESCRIPTION
**Pull request by TomPretty**
## Why are you doing this?
Implement the redesigned Thank You page as an ab test, as per [this card](https://trello.com/c/0nXhtGNV/1855-implement-the-new-thank-you-flow).

## Screenshots

mobile
Screenshot 2020-09-14 at 15 10 48

tablet
Screenshot 2020-09-14 at 15 11 02

desktop
Screenshot 2020-09-14 at 15 11 21

### CODE REVIEW

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    marketingConsentAction,
+    socialShareAction,
+    supportReminderAction,
+    surveyAction,
+    ausMapAction,
+  ];
+
+  const actions = countryId === 'AU' ? ausActions : defaultActions;
+
+  const shownComponents = actions
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx:193]: lovely

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYouAusMap.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import ActionBody from './components/ActionBody';
+import SvgAusMap from './components/SvgAusMap';
+import { OPHAN_COMPONENT_ID_AUS_MAP } from './utils/ophan';
+import { trackComponentClick } from 'helpers/tracking/behaviour';
+
+const buttonContainer = css`
+  margin-top: ${space[6]}px;
+`;
+
+const AUS_MAP_URL = 'https://support.theguardian.com/aus-2020-map';
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYouAusMap.jsx:18]: let's keep the tracking query string from the old thankyou page

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/utils/ophan.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    component: {
+      componentType: 'ACQUISITIONS_OTHER',
+    },
+    action: 'VIEW',
+    id: OPHAN_EVENT_ID_KNOWN_EMAIL,
+    value: isKnownEmail.toString(),
+  });
+};
+
+export const trackUserData = (
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/utils/ophan.js:78]: the old thank you page should also send this data

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYouAusMap.jsx (hunk 2 above)
Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYouAusMap.jsx:18]: good spot! 👍

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/utils/ophan.js (hunk 3 above)
Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/utils/ophan.js]: yeah good spot - just added this!

File: support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx (hunk 1 above)
Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx:193]: 😀

File: support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx [hunk 4]
```diff
@@ -26,8 +26,9 @@ import { init as formInit } from './contributionsLandingInit';
 import { initReducer } from './contributionsLandingReducer';
 import { ContributionFormContainer } from './components/ContributionFormContainer';
 import { enableOrDisableForm } from './checkoutFormIsSubmittableActions';
-import ContributionThankYouContainer
-  from './components/ContributionThankYou/ContributionThankYouContainer';
+import ContributionThankYou from './components/ContributionThankYou/ContributionThankYou';
+// import ContributionThankYouContainer
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx:30]: can be deleted

**Review by tomrf1 — APPROVED** [review]
brilliant work!

File: support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx (hunk 4 above)
Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx]: 👍

**Pull request merged**